deploy-manage/deploy/elastic-cloud/change-hardware.md

```diff
@@ -8,9 +8,11 @@ products:
   - id: cloud-hosted
 ---
 
-# Change hardware [ec-change-hardware-for-a-specific-resource]
+# Customize instance configuration [ec-change-instance-configuration]
```

Comment:
Changing the anchor name (the part under `[` and `]` could provoke existing links to this page (using the old anchor name) to fail.

Changing the heading name is not a problem, of course.

I would only change the anchor names if I was sure it's an anchor that hasn't been used or referenced in the past.